caddyhttp/httpserver/tplcontext.go

```diff
@@ -448,6 +449,20 @@ func (c Context) AddLink(link string) string {
 	return ""
 }
 
+// Returns TLS protocol version
+func (c Context) TLSVersion() string {
+	switch c.Req.TLS.Version {
```

Comment:
`c.Req.TLS` might be nil -- resulting in a panic.

Reply:
Good point. I'll add tests and fix this. Thnx!